QA_Android.md

```diff
@@ -0,0 +1,73 @@
+# Testing Energy Consumption on Android
+
+## Using Android Batterystats & Battery Historian
+
+Batterystats is a tool included in the Android framework that collects battery data on your device. Battery Historian converts the report from Batterystats into an HTML visualization that you can view in your browser.
+
+More info on [Batterystats & Battery Historian](https://developer.android.com/studio/profile/battery-historian.html).
+
+### Prerequisites
+
+- docker
+- git
+- [adb](https://developer.android.com/studio/command-line/adb.html) (included in [Android SDK](https://developer.android.com/studio/index.html) Platform-Tools package)
+- real Android device to measure battery consumption
+
+### Setup
+
+1. Install and run Battery Historian:
+```
+docker run -p 9999:9999 bhaavan/battery-historian
```

Comment:
Might want to also reference the authoritative source for battery-history, github.com/google/battery-historian , and suggest that if there are problems one should consider building from source.  This docker image was rebuilt "a year ago," and there was an update to the google repo last May, so it's not clear whether the image contains the latest code.